specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/examples/CopyModel.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "modelId": "{model Id}",
+    "body": {},
+    "copyRequest": {
+      "targetResourceId": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CognitiveServices/accounts/{resourceName}",
```

Comment:
Hello,
Salim Yachaoui from AiBuilder here,
Is this necessarry for you to have the full targetResourceId ? couldn't the endpoint+apikey suffice?

Reply:
or endpoint + resourceName ?

Reply:
The {apiKey} is PII. It is not advised to explicitly pass it around in the request. Besides, from an ALM perspective, you will have to store resource information to be able to use copy API. It is far easier and secure to store non-PII like AzureResourceId than to store endpoint + key.